pkg/ebpf/c/tracee.bpf.c

```diff
@@ -4360,19 +4363,22 @@ static __always_inline int tc_probe(struct __sk_buff *skb, bool ingress) {
     //
     // See bpf_skb_event_output in net/core/filter.c.
     u64 flags = BPF_F_CURRENT_CPU;
-    flags |= (u64)skb->len << 32;
-    if (get_config(CONFIG_OPTIONS) & OPT_DEBUG_NET){
+
+    // If not capture, only send the minimal required data to save the
```

Comment:
Why do you write "If not capture"? Shouldn't it be "if net_packet event wasn't chosen"?